### PR TITLE
Migrated blobfs smoke tests

### PR DESCRIPTION
**Pull request by gapra-msft**
test_blobfs_sas_download_1Kb_file -> Scenario_DownloadFile
test_blobfs_sas_download_64MB_file -> Scenario_DownloadFile64MB
test_blobfs_sas_download_100_1Kb_file -> Scenario_Download100Files

test_blobfs_download_1Kb_file -> Scenario_DownloadFile
test_blobfs_download_64MB_file -> skipped anyway
test_blobfs_download_100_1Kb_file -> Scenario_Download100Files
test_blobfs_download_200_1Kb_file -> probaby fine to skip? covered by 100 files anyway

test_blobfs_sas_upload_1Kb_file -> Scenario_UploadFile
test_blobfs_sas_upload_64MB_file -> Scenario_UploadFile64MB
test_blobfs_sas_upload_100_1kb_file -> Scenario_Upload100Files

test_blobfs_upload_1Kb_file_with_oauth -> Scenario_UploadFile
test_blobfs_upload_64MB_file_with_oauth ->  skipped anyway
test_blobfs_upload_100_1Kb_file_with_oauth -> Scenario_Upload100Files
test_blobfs_upload_uneven_multiflush_with_oauth -> Scenario_UploadFileUnevenMultiflushOAuth
test_blobfs_upload_even_multiflush_with_oauth -> Scenario_UploadFileEvenMultiflushOAuth

### CODE REVIEW

File: e2etest/zt_newe2e_blobfs_test.go [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+			},
+		},
+	})
+
+	ValidateResource[ObjectResourceManager](svm, dstContainer.GetObject(svm, fileName, common.EEntityType.File()), ResourceDefinitionObject{
+		Body: body,
+	}, true)
+}
+
+func (s *BlobFSTestSuite) Scenario_UploadFile64MB(svm *ScenarioVariationManager) {
```
Comment (adreed-msft) [inline e2etest/zt_newe2e_blobfs_test.go:41]: File size could be a variation, another valid question is: why are we doing a specific test of 64MB?

File: e2etest/zt_newe2e_blobfs_test.go [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+			ADLSFlushThreshold: pointerTo(uint32(15)),
+		},
+	})
+
+	ValidateResource[ObjectResourceManager](svm, dstContainer.GetObject(svm, fileName, common.EEntityType.File()), ResourceDefinitionObject{
+		Body: body,
+	}, true)
+}
+
+func (s *BlobFSTestSuite) Scenario_UploadFileEvenMultiflushOAuth(svm *ScenarioVariationManager) {
```
Comment (adreed-msft) [inline e2etest/zt_newe2e_blobfs_test.go:98]: Uneven/even could be a variation

File: e2etest/zt_newe2e_blobfs_test.go [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+			},
+		},
+	})
+
+	ValidateResource[ObjectResourceManager](svm, dstObj, ResourceDefinitionObject{
+		Body: body,
+	}, true)
+}
+
+func (s *BlobFSTestSuite) Scenario_DownloadFile64MB(svm *ScenarioVariationManager) {
```
Comment (adreed-msft) [inline e2etest/zt_newe2e_blobfs_test.go:187]: Ditto

File: e2etest/zt_newe2e_blobfs_test.go [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+			},
+		},
+	})
+
+	ValidateResource[ObjectResourceManager](svm, dstObj, ResourceDefinitionObject{
+		Body: body,
+	}, true)
+}
+
+func (s *BlobFSTestSuite) Scenario_Download100Files(svm *ScenarioVariationManager) {
```
Comment (adreed-msft) [inline e2etest/zt_newe2e_blobfs_test.go:215]: Up/Download could be varied over

**Review by adreed-msft — APPROVED** [review]
lgtm, could be compressed down, but otherwise fine.

File: e2etest/zt_newe2e_blobfs_test.go (hunk 3 above)
Comment (gapra-msft) [reply e2etest/zt_newe2e_blobfs_test.go:187]: resolved this

File: e2etest/zt_newe2e_blobfs_test.go (hunk 2 above)
Comment (gapra-msft) [reply e2etest/zt_newe2e_blobfs_test.go:98]: done

File: e2etest/zt_newe2e_blobfs_test.go (hunk 4 above)
Comment (gapra-msft) [reply e2etest/zt_newe2e_blobfs_test.go:215]: For up/download I think I will skip varying over mostly due to the account type needing to be manually specified for hns.

**Pull request merged**